compiler/src/iree/compiler/DispatchCreation/Passes.cpp

```diff
@@ -80,6 +80,14 @@ static llvm::cl::opt<bool> clEnableDataTiling(
                    "path, --iree-opt-data-tiling=false must be set as wells"),
     llvm::cl::init(false));
 
+// TODO(#20187): Enable and delete the option once the remaining work is landed
+// to main.
+static llvm::cl::opt<bool> clConvertToFlowEncodeOp(
```

Comment:
I think we are adding too many flags here. We need to consolidate all the data tiling related things under one flag.

Reply:
I was going to say that this flag can be avoided if I land my other ongoing PR. Good to know the concern, and I'll send the other one first!